examples/trials/benchmarking/automlbenchmark/nni/frameworks.yaml

```diff
@@ -0,0 +1,88 @@
+---
+
+NNI:
```

Comment:
It might be tricky to maintain NNI version 2.2 each release. Do you plan to freeze it forever?

Reply:
I have changed the flag to stable, which indicates that the latest stable release should be used. This flag does not influence any functionalities on our side. The automlbenchmark designed this flag for users to choose different versions of frameworks using command line options.